index.html

```diff
@@ -0,0 +1,22 @@
+!<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <title>Portfolio Webpage</title>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <link href="styles.css" rel="stylesheet">
+    </head>
+    <body>
+        <nav id="navbar" class="nav">
+            <a href="#projects"></a>
+        </nav>
+    <div id="welcome-section">
+        <h1>Welcome!</h1>
+    </div>
```

Comment:
Looking good 👍  but remember to use semantic tags such as `section` in place of your `div` s. Semantic tags make your code easier to read.